rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/synonyms/10_synonyms_put.yml

```diff
@@ -5,9 +5,6 @@ setup:
 
 ---
 "Create synonyms with no ID creates ID automatically":
-  - skip:
```

Comment:
Could you take all synonym changes and put them in a separate PR? They snuck in again maybe?